### PR TITLE
Return custom token

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -8,5 +8,7 @@ object Dependencies {
   lazy val httpComponents = "org.apache.httpcomponents" % "httpclient" % "4.5.11"
   lazy val jbossLogging = "org.jboss.logging" % "jboss-logging" % "3.4.1.Final"
   lazy val keycloakMock = "com.tngtech.keycloakmock" % "mock" % "0.2.0"
+  lazy val oauth2 = "com.nimbusds" % "oauth2-oidc-sdk" % "7.1.1"
+
```
Comment (TomJKing) [inline project/Dependencies.scala]: Unnecessary line breaks

File: src/main/scala/uk/gov/nationalarchives/tdr/keycloak/KeycloakUtils.scala [hunk 2]
```diff
@@ -23,6 +24,11 @@ class KeycloakUtils(url: String) {
         Option.empty
     }
   }
+
+  def token(token:String): Token = {
+    Token(getAccessToken(token), new BearerAccessToken(token))
+  }
+
```
Comment (TomJKing) [inline src/main/scala/uk/gov/nationalarchives/tdr/keycloak/KeycloakUtils.scala]: Unnecessary line break

File: src/test/scala/uk/gov/nationalarchives/tdr/keycloak/KeycloakUtilsTest.scala [hunk 3]
```diff
@@ -21,13 +21,55 @@ class KeycloakUtilsTest extends AnyFlatSpec with Matchers with BeforeAndAfterEac
...
+
+  "The token method " should "return the correct roles for a valid token" in {
+    val role = "role_admin"
+    val mockToken = mock.getAccessToken(aTokenConfig().withResourceRole("tdr", role).build())
+    val token = utils.token(mockToken)
+    token.roles.size should be(1)
+    token.roles should contain(role)
+  }
+
+  "The verifyToken method " should "return a bearer token for an invalid token string " in {
```
Comment (TomJKing) [inline src/test/scala/uk/gov/nationalarchives/tdr/keycloak/KeycloakUtilsTest.scala]: Should this read "The token method " ...

Comment (MancunianSam) [reply src/test/scala/uk/gov/nationalarchives/tdr/keycloak/KeycloakUtilsTest.scala]: It should indeed, thanks, I'll change it.

**Pull request merged**